### ISSUE TITLE
psutil error when installing dask

### ISSUE THREAD

**Issue by pgg1610** [dask/dask#6607]
**What happened**:
Trying to install dask-complete on miniconda python3.6.11 environment. Error when installing `dask` mostly related to psutil compilations. 
<img width="1918" alt="Screen Shot 2020-09-08 at 10 04 37 AM" src="https://user-images.githubusercontent.com/9443049/92487261-0697a000-f1bb-11ea-8f3d-7e5d80d6935c.png">
<img width="1899" alt="Screen Shot 2020-09-08 at 10 04 53 AM" src="https://user-images.githubusercontent.com/9443049/92487275-09929080-f1bb-11ea-8990-77addab1f71f.png">

Similar issue was seen elsewhere. [Link](https://github.com/tensorflow/datasets/pull/1057#issue-325217811) directs to the solution implemented in that case. 

**What you expected to happen**:
Get dask installed on the miniconda env. 

**Minimal Complete Verifiable Example**:

```python
pip install "dask[complete]"
```

**Anything else we need to know?**:

**Environment**:
- Dask version: N.A.  
- Python version: Main version 3.8.3, conda env version 3.6.11 
- Operating System: OS 10.15.6 
- Install method (conda, pip, source): `pip install "dask[complete]"`

**Comment by TomAugspurger** [comment]
Are you proposing that distributed drop psutil as a dependency? I'm not sure how feasible that is (though you're welcome to investigate).

I'd recommend using pip's `--only-binary=":all:"` if you don't have a C compiler in your environment, or use conda which doesn't require a compiler.

**Comment by jrbourbeau** [comment]
Thanks for opening an issue @pgg1610 

> Get dask installed on the miniconda env

Instead of using pip could you

**Comment by jrbourbeau** [comment]
Oops, accidentally hit the close button : )

@pgg1610 for reference you can install dask using conda with

```
conda install -c conda-forge dask
```

**Comment by pgg1610** [comment]
Thanks for getting back so quickly! I did try to install dask using conda route but there has been some issue with the conda since I run into following error: 
```bash 
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
```

For now, I will try re-configuring the conda-env and try installing dask again. Quick note: I did have dask installed and working in the previous conda-env (Python 3.4) but I have been facing issues ever since I switched to a new version.
